hardware-and-software-requirements.md

```diff
@@ -18,8 +18,8 @@ Starting from v6.1.1, TiDB provides multi-level support for different quality st
 
     |  Operating systems   |   Supported CPU architectures   |
     |   :---   |   :---   |
-    | Red Hat Enterprise Linux 8.4 or a later 8.x version  |  <ul><li>x86_64</li><li>ARM 64</li></ul>  |
-    | CentOS 8 Stream | <ul><li>x86_64</li><li>ARM 64</li></ul> |
+    | <ul><li>Red Hat Enterprise Linux 8.4 or a later 8.x version</li><li>CentOS 8.4 or a later 8.x version</li></ul>  |  <ul><li>x86_64</li><li>ARM 64</li></ul>  |
```

Comment:
There are two 'flavours' of CentOS 8:

- CentOS Linux 8, which has minor versions like 8.4. This has reached end-of-life. See also: https://www.centos.org/centos-linux-eol/
- CentOS Stream 8, which does not have minor versions. This is still supported upstream.

I don't think we should support a Linux distribution that is no longer supported upstream with security updates etc.